README.md

```diff
@@ -44,14 +44,13 @@ the accounts of `Alice` and `Bob`:
 ganache-cli --account="0x7d51a817ee07c3f28581c47a5072142193337fdca4d7911e58c5af2d03895d1a,100000000000000000000000" --account="0x6aeeb7f09e757baa9d3935a042c3d0d46a2eda19e9b676283dce4eaf32e29dc9,100000000000000000000000"
 ```
 
-In a second and third terminal, cd inside folder `perun` of `go-perun` and start
-the nodes of Alice and Bob with
+In a second and third terminal, start the nodes of Alice and Bob with
 ```sh
-./perun-eth-demo main.go demo --config alice.yaml
+./perun-eth-demo demo --config alice.yaml
 ```
 and
 ```sh
-./perun-eth-demo main.go demo --config bob.yaml
+./perun-eth-demo demo --config bob.yaml
```

Comment:
Did you go through the example and try it out? Does it still all work?

Reply:
yes, went through it, it works. (if you include "main.go", it fails with `Error: unknown command "main.go" for "perun"`.)

Reply:
we could also add `--network network.yaml` to explicitly demonstrate how the network is configured. what do you think?

Reply:
Is the network currently implicitly loaded from this file? If yes, I like this idea.

Reply:
This is already done. See `demo_test.go:TestNodes` for reference.

Reply:
yeah, but it would be instructive for the reader to have this explicitly in the walkthrough. anyways, i don't think it is that important and we can leave as is for now